### ISSUE TITLE
⛔ :link: ♿ 🐞 "Wordpress menu" widget is adding background color by default

### ISSUE THREAD

**Issue by dieram-dev** [elementor/elementor#24239]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The "Wordpress menu" widget is adding a background color by default on ".elementor-nav-menu--dropdown .elementor-item:focus", there is no way to change it on the settings panel other than custom coding a css overwrite.
<img width="311" alt="Screen Shot 2023-11-10 at 08 39 02" src="https://github.com/elementor/elementor/assets/98188678/3de34493-2f86-41a0-b922-07f35e3f8998">
<img width="389" alt="Screen Shot 2023-11-10 at 08 39 12" src="https://github.com/elementor/elementor/assets/98188678/9e610727-a1b4-40cf-a253-be2b948e138b">

### Steps to reproduce

1) Add "Wordpress menu" widget and select an already created menu
2) Change some of the styling for previewing purpose
3) Update changes
4) Preview the updated page
5) Click on the menu items and notice how a background color appears on the items when clicked (it is easier to reproduce if the menu item doesn't have a link, but an anchor instead "#")

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.5.22
	PHP Version: 7.4.33
	PHP Memory Limit: 768M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.1
	Site URL: https://soulrevolutionfestival.getyoursite.today
	Home URL: https://soulrevolutionfestival.getyoursite.today
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 768M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36

== Active Plugins ==
	Disable Gutenberg
		Version: 3.1
		Author: Jeff Starr

	Elementor
		Version: 3.16.5
		Author: Elementor.com

	Elementor Pro
		Version: 3.16.2
		Author: Elementor.com

	Envato Elements
		Version: 2.0.12
		Author: Envato


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active by default
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


== Log ==
	
PHP: showing 3 of 3PHP: 2023-10-17 00:44:29 [notice X 7][../wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-10-17 01:13:54 [notice X 10][../wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-11-03 19:12:56 [notice X 1][../wp-content/plugins/elementor/includes/widgets/image-carousel.php::851] Undefined index: id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 4 of 4JS: 2023-10-30 22:25:08 [error X 9][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.16.5:3:1109735] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-10-30 22:28:41 [error X 4][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.16.5:3:54034] Cannot read properties of undefined (reading 'activate') 
JS: 2023-10-31 20:28:24 [error X 2][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.16.5:3:639892] Cannot read properties of undefined (reading 'isDesignable') 
JS: 2023-10-31 21:02:38 [error X 2][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.16.5:3:689303] this.model.isValidChild is not a function  



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	Envato Elements: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	Envato Elements: Incompatible
				

````

</details>

**Comment by nicholaszein** [comment]
Hello there @dieram-dev!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.29.0! 🥳 

Now, the color selected for the `:hover` state is also applied to the `:focus` state of the submenus in the WordPress Menu widget!

Thank you for bringing this to our attention! 🙏 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
